react-state3/react-state-3_journal-app-form/src/components/EntryForm/index.js

```diff
@@ -1,18 +1,57 @@
 import "./EntryForm.css";
 import Button from "../Button";
+import { useState } from "react";
+
+export default function EntryForm({ onAddEntry }) {
+  const [formData, setFormData] = useState({ motto: "", notes: "" });
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    // Create a new entry object with the values from the form
+    const newEntry = {
+      motto: formData.motto,
+      notes: formData.notes,
+    };
```

Comment:
Interesting solution! Alternatively, you can get the form data with `FormData()` ( as in the instructions). Then the code is smth like this: 
 ```
const formData = new FormData(event.target); 
const data = Object.fromEntries(formData); 
onAddEntry(data);
```


If you use the `FormData()`,  you would need neither the second sate nor the function `handleInputChange`